easybuild/easyconfigs/g/gretl/gretl-2021c-foss-2020b.eb

```
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Loris Bennett
+# ZEDAT, Freie Universitaet Berlin
+
+easyblock = 'ConfigureMake'
+
+name = 'gretl'
+version = '2021c'
+
+homepage = 'http://gretl.sourceforge.net'
+description = "A cross-platform software package for econometric analysis"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+# http://prdownloads.sourceforge.net/gretl/gretl-2020a.tar.xz
+source_urls = ['http://prdownloads.sourceforge.net/%(name)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['be468b8233b2702a15df24d1efffdf44ce082c99617cbfa01d8e2fafda370f8a']
+
+builddependencies = [
+    ('gnuplot', '5.4.1'),
+    ('libxml2', '2.9.10'),
+    ('GMP', '6.2.0'),
+    ('GtkSourceView', '3.24.11'),
+    ('ImageMagick', '7.0.10-35'),
+    ('MPFR', '4.1.0'),
+]
+
+
+preconfigopts = 'export LDFLAGS="-fopenmp $LDFLAGS" && '
+
+configopts = '--disable-json '
+
+sanity_check_paths = {
+    'files': ['bin/gretl', 'bin/gretlcli'],
+    'dirs': ['bin', 'share']
```

And, now that MPI support has been enabled again, it would be good to also add the MPI binaries to the sanity check again:

```suggestion
    'files': ['bin/gretl', 'bin/gretlcli', 'bin/gretlmpi', 'lib/libgretl-1.0.%s' % SHLIB_EXT],
    'dirs': ['include/gretl', 'share']
```